packer/nomad-server/image.pkr.hcl

```diff
@@ -0,0 +1,163 @@
+packer {
+  required_version = ">= 0.12.0"
+}
+
+variable "build_ami" {
+  description = "Whether or not to actually build an AMI. Set to `false` for doing testing"
+  type        = bool
+  default     = true
+}
+
+variable "ami_name_prefix" {
+  type    = string
+  default = "nomad-server"
+}
+
+variable "aws_region" {
+  type    = string
+  default = "us-east-1"
+}
+
+variable "consul_module_version" {
+  type    = string
+  default = "v0.10.1"
+}
+
+variable "consul_version" {
+  type    = string
+  default = "1.9.6"
+}
+
+variable "nomad_version" {
+  type    = string
+  default = "1.1.1"
+}
+
+variable "terraform_aws_nomad_version" {
+  type    = string
+  default = "v0.9.1"
+}
+
+variable "git_sha" {
+  description = "The git SHA of the commit the AMI is being generated from. If present, will be used to tag the AMI."
+  type        = string
+  default     = env("GIT_SHA")
+}
+
+variable "git_branch" {
+  description = "The git branch the AMI is being generated from. If present, will be used to tag the AMI."
+  type        = string
+  default     = env("GIT_BRANCH")
+}
+
+variable "buildkite_build_number" {
+  description = "The build number of the Buildkite pipeline this AMI was built in. If present, will be used to tag the AMI."
+  type        = string
+  default     = env("BUILDKITE_BUILD_NUMBER")
+}
```

Comment:
Might be valuable to extract these variables into a separate variable file and source that in both builds, particularly if all the Consul and Nomad versions are expected to be the same.

Reply:
since merged, no longer worth it